CHANGELOG.md

```diff
@@ -1,7 +1,79 @@
+
+![Persons New UX](https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/persons-v2.png)
+
+The UI of our person pages just got a whole lot better! As is often the case with our larger features, this isn't news to all of you. We had this behind a feature flag and have now decided to roll it out for everyone.
 
+Now you can visualize user properties alongside a user's events, and most of the context you need on a person is available to you in a sleek UI without you needing to scroll.
+
+Oh, and the code got much better as a result too...
+- [Exposing $set and $set_once on all events](https://github.com/PostHog/posthog/pull/3363)
+
+![Set properties on any event](https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/%24set.png)
+
+The event properties `$set` and `$set_once` can now be used on any event to set properties directly to the user associated with that event.
+
+Previously, this would only work on `$identify` events, making it so that you needed to call multiple methods in order to send an event and set user properties based on the same data. But now, you can do it all in one, as shown in the image above.
+
+- [Event Sequence Timer Plugin](https://posthog.com/plugins/event-sequence-timer-plugin)
+
+![Event Sequence Timer Plugin](https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/time-events.png)
+
+Our users requested a way to measure the time passed between certain events, and this is it!
+
+By installing the Event Sequence Timer Plugin, you can specify as many sets of events as you want and the plugin will track the time between them, either using a first touch or last touch mechanism.
+
+It will then add a property to your events that allows you to easily build visualizations in PostHog of the average, minimum, and maximum time between events, as well as all the other mathematical operations we support.
+
+- [Property Flattener Plugin](https://posthog.com/plugins/property-flattener)
+
+![Property Flattener Plugin](https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/flattener.png)
+
+The new Property Flattener Plugin allows you to convert event properties contained in a nested structure into a flat structure, allowing you to set filters based on the nested properties.
+
+
+- [Project API Key Autofill in Docs for Cloud Users](https://github.com/PostHog/posthog.com/pull/998)
+
+![Docs Token Autofill](https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/token-autofill.png)
+
+If you're a user of PostHog Cloud, we now autofill your Project API Key and API Host automatically in the Docs for you, meaning you can copy-paste snippets and use them directly with no manual changes!
+
+This key will be based on the last project you used in PostHog, and you can check what project that is by simply hovering your cursor over the highlighted key.
+### 1.21.0 - Wednesday 17 February 2021
 
```

Comment:
#2952 is a pretty major feature that's still behind a feature flag. not sure if these are included in changelog or not

Reply:
No, those behind feature flags don't go in - There's quite a few big features that remain unreleased: funnel trends, formulas, cohorts on modal, labels

Reply:
Will make for a nice 1.23.0 :smiley: